src/main/webapp/EditHelp.js

```diff
@@ -0,0 +1,100 @@
+Behaviour.specify("A.help-button", 'hudson-behavior', 1, function(e) {
```

Comment:
You should try to use modular form of the file and split functionality to modules
https://github.com/jenkinsci/github-plugin/blob/master/src/main/webapp/js/warning.js
https://github.com/jenkinsci/github-plugin/blob/master/src/main/resources/com/cloudbees/jenkins/GitHubPushTrigger/config.groovy#L14-L18